src/middle/Mir.ml

```diff
@@ -100,13 +100,13 @@ and 'e expr =
   | Indexed of 'e * 'e index list
 [@@deriving sexp, hash, map, compare, fold]
 
-type fun_arg_decl = (autodifftype * string * unsizedtype) list
+type fun_arg_decl = autodifftype * string * unsizedtype
```

Comment:
I agree with the change but is it making a separate PR?

Reply:
Is this a big change that makes the rest of this PR hard to review?

Reply:
Not now the Mir refactor is in!